GTMSessionFetcher.podspec

```diff
@@ -2,7 +2,7 @@
 # to import GTMSessionFetcher via the CocoaPods dependency Manager.
 Pod::Spec.new do |s|
   s.name        = 'GTMSessionFetcher'
-  s.version     = '1.5.0'
+  s.version     = '1.6.0'
```

Comment:
Please don't bump the version number in this PR. We like to keep that separate from individual code changes.